reviewdog.go

```diff
@@ -20,11 +20,15 @@ type Reviewdog struct {
 	d DiffService
 }
 
+var ExportedVarWithoutComment = 1 // TODO(#20): revert
```

Comment:
:dog:  [[reviewdog](https://github.com/haya14busa/reviewdog)] :dog:
exported var ExportedVarWithoutComment should have comment or be unexported